_posts/2023-09-07-develop-functions-locally.md

```diff
@@ -0,0 +1,278 @@
+---
+title: "The faster way to iterate on your OpenFaaS functions"
+description: "Learn how the local-run and watch features of faas-cli can help you iterate on functions faster."
+date: 2023-09-07
+categories:
+- faas-cli
+- openfaas
+- functions
+- devex
+dark_background: true
+author_staff_member: han
+image: "/images/2023-09-develop-functions-locally/background.png"
+hide_header_image: true
+---
+
+Learn how the local-run and watch features of faas-cli can help you iterate on functions faster.
+
+This is what the typical development lifecycle of an OpenFaaS function looks like:
+
+- Make code changes
+- Build the function image
+- Push the image to a container registry
+- Deploy the function
```

Comment:
Missing a step here.

- Wait for the new image to be pulled and to become ready